docs/resources.md

```diff
@@ -85,7 +84,9 @@ Generic Attestation Policy Spec description:
 
 | Field     | Default (if applicable)   | Description |
 |-----------|---------------------------|-------------|
-| attestationAuthorityNames | | List of [Attestation Authorities](#attestationauthority-crd) for which one of is required to be satisfied before the Admission Controller will admit the pod.|
+| attestationAuthorityNames | | Non-empty List of [Attestation Authorities](#attestationauthority-crd) for which all of them are required to be satisfied before the Admission Controller will admit the pod.|
+
+Note that the list of [Attestation Authorities](#attestationauthority-crd) must be non-empty. If the list is empty, images can only be admitted through allowlist (GAP allowlist is WIP).
```

Comment:
Mention that evaluating a policy with an empty aa list results in an error.

Reply:
Come to think of it, we should both understand and document the error behavior.  (Probably depends on the webhook config.)

Nothing to do with this PR though...

Reply:
I thought about saying "results in an error", then I thought it would exclude the case for allowlisted images + 0 AA gap. Be more explicit for the non-allowlisted case?

What other behaviors and interactions with webhook config are you thinking?

Reply:
PTAL at my comment in review.go.  I think that "0 AAs in GAP" should always be an error, regardless of the request being checked.

Reply:
Done.